index.js

```diff
@@ -20,6 +21,19 @@ app.engine('hbs', hbs({ // HBS Config
   defaultView: 'default',
   layoutsDir: __dirname + '/views/layouts/',
   partialsDir: __dirname + '/views/partials/',
+  helpers: {
+    count: function(array){
+      var count = 0;
+      if (array.length > 0)
+      {
+        for (var i = 0; i < array.length; i++)
+        {
+          count++;
+        }
+      }
+      return count;
+  },
+}
```

Comment:
This can be a mongoose.countDocuments()

Please see: https://mongoosejs.com/docs/api.html#model_Model.countDocuments